lib/virtual_device.js

```diff
@@ -121,6 +121,27 @@ VirtualDevice.prototype.call = function(/* transition, args, cb */) {
 
 };
 
+VirtualDevice.prototype.available = function(transition) {
+  return !!this._getAction(transition);
+};
+
+VirtualDevice.prototype.transitionsAvailable = function() {
```

Comment:
Are we actually using `transitionsAvailable()` or `properties()` anywhere?  I don't think we recommend using these in the JavaScript API for local devices.  If that's the case, I'd say we get rid of 'em in VirtualDevice for now.